CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
Do we have someone using that version of CMake who can confirm it works? Maybe there is a CI with this version we can try?